apps/web/src/pages/ManagerDashboardPage/Components/ReviewTalentRequestDialog.tsx

```diff
@@ -0,0 +1,461 @@
+          >
+            {classifications.length > 0
+              ? deriveSingleString(
+                  classifications,
+                  formatClassificationString,
+                  locale,
+                )
+              : nullMessage}
+          </FieldDisplay>
+
+          <FieldDisplay
+            label={intl.formatMessage(talentRequestMessages.jobTitle)}
+          >
+            {request.jobTitle ?? nullMessage}
+          </FieldDisplay>
+
+          <FieldDisplay
+            label={intl.formatMessage(talentRequestMessages.workStream)}
+          >
+            {workStreams.length > 0
+              ? deriveSingleString(
+                  workStreams,
+                  (stream) => getLocalizedName(stream.label, intl),
+                  locale,
+                )
+              : nullMessage}
+          </FieldDisplay>
+
+          <FieldDisplay
+            label={intl.formatMessage(talentRequestMessages.languageProfile)}
+          >
+            {request.applicantFilter?.languageAbility
+              ? getLocalizedName(
+                  request.applicantFilter.languageAbility.label,
+                  intl,
+                )
+              : nullMessage}
+          </FieldDisplay>
+
+          <FieldDisplay
+            data-h2-grid-column="p-tablet(span 2)"
+            label={intl.formatMessage(talentRequestMessages.supervisoryStatus)}
+          >
+            {request.positionType
+              ? positionTypeToYesNoSupervisoryStatement(
+                  request.positionType.value,
+                  intl,
+                )
+              : nullMessage}
+          </FieldDisplay>
+
+          <FieldDisplay
+            data-h2-grid-column="p-tablet(span 2)"
+            label={intl.formatMessage(talentRequestMessages.employmentDuration)}
+          >
+            {
+              // has its own null state
+              intl.formatMessage(
+                getEmploymentDuration(
+                  positionDurationToEmploymentDuration(
+                    unpackMaybes(request.applicantFilter?.positionDuration),
+                  ),
+                ),
+              )
+            }
+          </FieldDisplay>
+
+          <FieldDisplay
+            data-h2-grid-column="p-tablet(span 2)"
+            label={intl.formatMessage(
+              talentRequestMessages.educationRequirement,
+            )}
+          >
+            {
+              // has its own null state
+              hasDiplomaToEducationLevel(
+                request.applicantFilter?.hasDiploma,
+                intl,
+              )
+            }
+          </FieldDisplay>
+        </div>
+        <Separator orientation="horizontal" data-h2-margin="base(0)" />
+        <div
+          data-h2-display="base(flex)"
+          data-h2-flex-direction="base(column)"
+          data-h2-gap="base(x0.5)" // accordions have x0.5 margin built-in already
+        >
+          <Accordion.Root type="single" collapsible data-h2-margin="base(0)">
```

Comment:
You can move all the accordion items under one Root and make a group. (Currently each accordion in the dialog has it's own Root)

Reply:
It makes sense to me but I'm not sure that's how the design has them.  You can see each collapsible is at the same nesting level as the neighboring elements and the spacing is noticeably larger than our accordion group. 
![image](https://github.com/user-attachments/assets/d25c4847-0fd2-4946-bf3d-51630c7f139c)

@JoshBeveridge can you confirm?

Reply:
@petertgiles I hadn't used a group here, but there are standards outlined in the accordion component Figma page for how groups should be structured and spaced. Best to follow that. Let me know if it diverges drastically from what you're seeing in this dialog and I'll take another look.

Reply:
OK, sounds good, I'll switch it over.  The accordion group is just spaced a bit tighter.
![image](https://github.com/user-attachments/assets/6388133f-02d4-43d6-988a-73408130ad47)

Reply:
[switch to accordion group](https://github.com/GCTC-NTGC/gc-digital-talent/pull/11653/commits/285d31e2c10138acd143a343f5851ab18715b621)